vaadin-card-flow-parent/vaadin-card-testbench/src/main/java/com/vaadin/flow/component/card/testbench/CardElement.java

```diff
@@ -23,4 +29,102 @@
  */
 @Element("vaadin-card")
 public class CardElement extends TestBenchElement {
+
+    /**
+     * Gets the content elements.
+     *
+     * @return the content elements
+     */
+    public List<TestBenchElement> getContents() {
+        try {
+            var contentWrapper = findElement(By.cssSelector("div:not([slot])"));
+            return contentWrapper.getPropertyElements("children");
+        } catch (NoSuchElementException e) {
+            return Collections.emptyList();
+        }
```

Comment:
```suggestion
        return findElements(By.cssSelector("div:not([slot])")).stream()
                .findFirst()
                .map(wrapper -> ((TestBenchElement) wrapper)
                        .getPropertyElements("children"))
                .orElse(Collections.emptyList());
```
Might be nitpicking, but it seems weird to force an exception when there's an alternative API that works without it. The JavaDoc for `WebElement.findElement` also recommends to use `WebElement.findElements`.